extensions/community_extensions/dsa_transparency.md

```diff
@@ -0,0 +1,379 @@
+                    "params": [1]},
+                    {"domain": "SSP2domain.com",
+                    "params": [1,2]
+                    }]
+            }
+        }
+    },
+    "imp": [{
+        "id": "1"
+    }],
+    "site": {
+        "id": "102855"
+    },
+    "user": {
+        "id": "55816b39711f9b5acf3b90e313ed29e51665623f"
+    }
+}
+
+```
+
+<h3>Bid Response</h3>
+<h4>Object: Bid</h4>
+<div>
+<table>
+<tbody>
+<tr>
+<td><strong>Attribute</strong></td>
+<td><strong>Type</strong></td>
+<td><strong>Description</strong></td>
+</tr>
+<tr>
+<td><code>ext.dsa</code></td>
+<td>object&nbsp;</td>
+<td>DSA Ad Transparency information</td>
+</tr>
+</tbody>
+</table>
+</div>
+<h4>Object: DSA</h4>
+<div>
+<table>
+<tbody>
+<tr>
+<td><strong>Attribute</strong></td>
+<td><strong>Type</strong></td>
+<td><strong>Description</strong></td>
+</tr>
+<tr>
+<td><code>behalf</code></td>
+<td>string</td>
+<td>Advertiser Transparency: Free UNICODE text string with a name of whose behalf the ad is displayed. Maximum 100 characters.</td>
+</tr>
+<tr>
+<td><code>paid</code></td>
+<td>string</td>
+<td>Advertiser Transparency: Free UNICODE text string of who paid for the ad. Must always be included even if it's the same as what is listed in the behalf attribute. Maximum 100 characters</td>
+</tr>
+<tr>
+<td><code>transparency</code></td>
+<td>array of object</td>
+<td>Array of objects of the entities that applied user parameters and the parameters they applied.</td>
+</tr>
+<tr>
+<td><code>adrender</code></td>
+<td>integer</td>
+<td>Flag to indicate that buyer/advertiser will render their own DSA transparency information inside the creative.
+    <p><code>0</code> = buyer/advertiser will not render</p>
+    <p><code>1</code> = buyer/advertiser will render</p></td>
+</tr>
+</tbody>
+</table>
+</div>
+<h4>Object: Transparency</h4>
+<div>
+<table>
+<tbody>
+<tr>
+<td><strong>Attribute</strong></td>
+<td><strong>Type</strong></td>
+<td><strong>Description</strong></td>
+</tr>
+<tr>
+<td><code>domain</code></td>
+<td>string</td>
+<td>Domain of the entity that applied user parameters&nbsp;</td>
+</tr>
+<tr>
+<td><code>params</code></td>
+<td>array of integer</td>
+<td>Array of buy-side applied user parameter targeting (using <a href="#user_parameters" target="_blank" rel="noopener">the list provided by DSA Transparency Taskforce</a>). Include support for multiple vendors who may add their own user-targeting parameters.</td>
+</tr>
+</tbody>
+</table>
+</div>
+<h4>Sample&nbsp; OpenRTB 2.6 Bid Response with DSA transparency:</h4>
+
+```
+{
+    "id": "1234567890",
+    "bidid": "abc1123",
+    "seatbid": [{
+   	 "seat": "512",
+   	 "bid": [{
+   		 "id": "1",
+   		 "nurl": "http://adserver.com/winnotice?impid=102",
+   		 "iurl": "http://adserver.com/pathtosampleimage",
+   		 "adomain": [
+   			 "advertiserdomain.com"
+   		 ],
+   		 "ext": {
+   			 "dsa": {
+   				 "behalf": "Advertiser",
+   				 "paid": "Advertiser",
+   				 "transparency": {
```

Comment:
I think it would be a good idea to change transparency in the example to array of objects instead of an object, as it's defined in specification as "array of object".